### PR TITLE
[10.0] Add module l10n_fr_hr_check_ssnid

### PR DESCRIPTION
**Pull request by alexis-via**
See README for more details about this new module.

### CODE REVIEW

**Comment by feketemihai** [comment]
@alexis-via You ca check python-stdnum on France you have "nir" validation...

**Comment by bealdav** [comment]
yes https://github.com/arthurdejong/python-stdnum/blob/master/stdnum/fr/nir.py

File: l10n_fr_hr_check_ssnid/views/hr_employee.xml [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+
+<odoo>
+
+
+<record id="view_employee_form" model="ir.ui.view">
+    <field name="model">hr.employee</field>
+    <field name="inherit_id" ref="hr.view_employee_form" />
+    <field name="arch" type="xml">
+        <field name="identification_id" position="after">
+            <field name="ssnid" groups="hr.group_hr_user"/>
```
Comment (legalsylvain) [inline l10n_fr_hr_check_ssnid/views/hr_employee.xml:15]: Hehe. always funny to have to add in OCA views, fields created by Odoo SA, but never used.

**Review by legalsylvain — APPROVED** [review]
:+1: Thanks. LGTM. Code review / no test.

> @alexis-via You ca check python-stdnum on France you have "nir" validation...

indeed, using the python lib could make the code lighter.

File: l10n_fr_hr_check_ssnid/models/hr_employee.py [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+class HrEmployee(models.Model):
+    _inherit = 'hr.employee'
+
+    @api.constrains('ssnid')
+    def france_ssnid_constrain(self):
+        for empl in self:
+            if empl.company_id.country_id.code == 'FR' and empl.ssnid:
+                try:
+                    validate(empl.ssnid)
+                except Exception, e:
```
Comment (legalsylvain) [inline l10n_fr_hr_check_ssnid/models/hr_employee.py]: Hi, 
thanks for using this lib ! 
Just one thing. Your current try / catch catches all exceptions. could you be more precise, by using : 

``except (InvalidFormat, InvalidLength, InvalidChecksum) as e:``

**Comment by alexis-via** [comment]
Travis is now green, so it's ready to merge.

**Review by bealdav — APPROVED** [review]

**Comment by legalsylvain** [comment]
Thanks @alexis-via. usefull module !

**Pull request merged**